architecture/core_concepts/builds_and_image_streams.adoc

```diff
@@ -170,8 +170,9 @@ endif::[]
 == Image Streams
 
 An _image stream_ comprises any number of
-xref:containers_and_images.adoc#docker-images[Docker-formatted container images] identified by tags.
-It presents a single virtual view of related images, similar to an image repository, and may contain images from any of the following:
+xref:containers_and_images.adoc#docker-images[Docker-formatted container images]
+identified by tags. It presents a single virtual view of related images, similar
```

Comment:
Need to update this path:
xref:../architecture/core_concepts/containers_and_images.adoc#docker-images[Docker-formatted container images]